pkg/webhook/azuredevops/webhook.go

```diff
@@ -0,0 +1,110 @@
+// File copied from https://github.com/go-playground/webhooks/blob/master/azuredevops/azuredevops.go
```

Comment:
nit: I think it could make sense to copy the corresponding test file over as well, just to be on the safe side. How do you see it?

Reply:
we would also need to copy all the testdata files.. https://github.com/go-playground/webhooks/blob/master/azuredevops/azuredevops_test.go#L63

I would avoid copying all those files. Should we leave it without the test or maybe we can just transfer my fork to the fleet org and use the fork instead?